watcher/src/watcher_db.rs

```diff
@@ -147,6 +150,8 @@ impl WatcherDB {
             Environment::new()
                 .set_max_dbs(10)
                 .set_map_size(MAX_LMDB_FILE_SIZE)
+                // TODO - needed because currently our test cloud machines have slow disks.
+                .set_flags(EnvironmentFlags::NO_SYNC)
```

Comment:
What are the implications of this? Is sync only related to writing or does it mean a read-only handle might read in partial data?

Reply:
The implications is that `lmdb` will not explicitly call `fsync` after every write, letting the OS handle it. We do the exact same thing for the ledger db and it works okay. We could theoretically lose some data this way but when the watcher is restarted it will pick up where it left (based on whats in the db)